addons/docs/src/frameworks/svelte/svelte-docgen-loader.ts

```diff
@@ -1,7 +1,36 @@
 import svelteDoc from 'sveltedoc-parser';
-
 import * as path from 'path';
 
+// From svelte/compiler/compile/utils but not exported
```

Comment:
Let's do this:
```suggestion
// From https://github.com/sveltejs/svelte/blob/8db3e8d0297e052556f0b6dde310ef6e197b8d18/src/compiler/compile/utils/get_name_from_filename.ts
// Copied because it is not exported from the compiler
```